### PR TITLE
[MM-28212] Upgrade Mattermost Cloud sales card

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
Sales card to upsell Mattermost Cloud subscriptions to free users.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28212

### CODE REVIEW

**Review by stevemudie — APPROVED** [review]
LGTM - Will test once merged and able to see complete package

File: components/admin_console/billing/billing_subscriptions.tsx
Comment (jwilander) [inline components/admin_console/billing/billing_subscriptions.tsx:135]: Should we remove the commented out code?

**Review by matthewbirtch — APPROVED** [review]
Looks so good @devinbinnie. Nice work!

**Review by jwilander — APPROVED** [review]

File: images/upgrade-mattermost-cloud-image.svg
Comment (marianunez) [inline images/upgrade-mattermost-cloud-image.svg:1]: Do we need to wrap these in components to make the svg theme compatible?

**Review by marianunez — APPROVED** [review]

File: images/upgrade-mattermost-cloud-image.svg
Comment (devinbinnie) [reply images/upgrade-mattermost-cloud-image.svg:1]: Shouldn't be an issue as this is only on display in the System Console, which isn't themeable afaik.

**Pull request merged**
